charter/index.html

```diff
@@ -433,30 +435,30 @@ <h3 id="w3c-coordination">W3C Groups</h3>
             <dd>To maintain a separation of goals and transfer of incubated work items.</dd>
 
             <dt><a href="https://www.w3.org/groups/cg/unhosted">Unhosted Web Community Group</a></dt>
-            <dd>To explore the overlaps (CORS, user-centric storage, JSON-LD container descriptions) and differences (OAuth instead of WAC/ACP) between the Solid Protocol draft and their work on <a href="https://datatracker.ietf.org/doc/draft-dejong-remotestorage/">remoteStorage</a>.</dd>
+            <dd>To explore the overlaps (CORS, user-centric storage, JSON-LD container descriptions) and differences (OAuth instead of WAC/ACP) between the PUMPKIN Protocol draft and their work on <a href="https://datatracker.ietf.org/doc/draft-dejong-remotestorage/">remoteStorage</a>.</dd>
 
             <dt><a href="https://www.w3.org/groups/wg/vc">Verifiable Credentials Working Group</a></dt>
-            <dd>To inquire on the suitability of credentials data models and serializations, and their potential integration with the Solid Protocol.</dd>
+            <dd>To inquire on the suitability of credential data models and serializations, and their potential integration with the PUMPKIN Protocol.</dd>
 
             <dt><a href="https://www.w3.org/groups/cg/webid">WebID Community Group</a></dt>
-            <dd>To provide integration feedback on the use of WebIDs in Solid Protocol (and related technical reports) in order to stabilise the fundamental notions and behaviors.</dd>
+            <dd>To provide integration feedback on the use of WebIDs in PUMPKIN Protocol (and related technical reports) in order to stabilize the fundamental notions and behaviors.</dd>
```

Comment:
```suggestion
            <dd>To provide integration feedback on the use of WebIDs in PUMPKIN Protocol (and related technical reports) to stabilize the fundamental notions and behaviors.</dd>
```

Reply:
I'd rather keep "in order to", which makes it easier for me to parse the relation between "to provide" and "to stabilize".